src/jit/emitinl.h

```diff
@@ -388,6 +421,35 @@ ssize_t             emitter::emitGetInsAmdAny(instrDesc *id)
                              regmask |= RBM_R11;
 #endif
 
+#ifdef _TARGET_ARM64_
+    assert(REGNUM_BITS >= 5);
+    encodeMask = id->idReg1();
+
+    if (encodeMask & 0x01)
+        regmask |= RBM_R19;
+    if (encodeMask & 0x02)
+        regmask |= RBM_R20;
+    if (encodeMask & 0x04)
+        regmask |= RBM_R21;
+    if (encodeMask & 0x08)
+        regmask |= RBM_R22;
+    if (encodeMask & 0x10)
+        regmask |= RBM_R23;
+
+    encodeMask = id->idReg2();
+
+    if (encodeMask & 0x01)
+        regmask |= RBM_R24;
+    if (encodeMask & 0x02)
+        regmask |= RBM_R25;
+    if (encodeMask & 0x04)
+        regmask |= RBM_R26;
+    if (encodeMask & 0x08)
+        regmask |= RBM_R27;
+    if (encodeMask & 0x10)
+        regmask |= RBM_R28;
+#endif
```

Comment:
I'm not sure what you meant. Do you mean to factor out L428-L437 for L441-L450?
If that is the case, I need to pass a range of register as parameters like R19-R23 vs. R24-R28. But this is about callee save registers so I'd like them to be explicitly listed like this (same as other architecture) instead of hiding them in a separate method.